### PR TITLE
Person: Polish documentation and custom variants

### PR DESCRIPTION
**Pull request by lynamemi**
#### Pull request checklist

- [ ] Addresses an existing issue: Fixes #0000
- [X] Include a change request file using `$ npm run change`

#### Description of changes

- Improved spacing between examples.
- Added DirectionalHint for Tooltip so that it can handle wide elements.

# **BEFORE:**
image
image
image

# **AFTER:**
image
image
image

#### Focus areas to test

(optional)

### CODE REVIEW

**Comment by micahgodbolt** [comment]
I seriously read this as "Polish language docs"

File: packages/office-ui-fabric-react/src/components/Persona/Persona.tsx [hunk 1]
```diff
@@ -122,7 +123,13 @@ export class Persona extends BaseComponent<IPersonaProps, {}> {
       <div className={ className }>
         { render
           ? render(this.props)
-          : <TooltipHost content={ text } overflowMode={ TooltipOverflowMode.Parent }>{ text }</TooltipHost>
+          : <TooltipHost
+            content={ text }
+            overflowMode={ TooltipOverflowMode.Parent }
+            directionalHint={ DirectionalHint.topLeftEdge }
```
Comment (mtennoe) [inline packages/office-ui-fabric-react/src/components/Persona/Persona.tsx]: Does this work in RTL?

File: packages/office-ui-fabric-react/src/components/Persona/Persona.scss [hunk 2]
```diff
@@ -219,6 +220,17 @@ $ms-Persona-presenceBorder: 2px;
   }
 }
 
+.root.coinSizeWithPresence {
+  .presence {
+    height: $ms-Persona-presenceSizeFlex;
+    width: $ms-Persona-presenceSizeFlex;
+
+    .presenceIcon {
+      display: none;
```
Comment (mtennoe) [inline packages/office-ui-fabric-react/src/components/Persona/Persona.scss]: What if the custom size is big so there is actually room for the presenceicon?

File: common/changes/office-ui-fabric-react/person-edge-polish_2017-11-06-16-27.json [hunk 3]
```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "office-ui-fabric-react",
+      "comment": "Persona: Flex sizing for presence with custom coinSize, DirectionalHint for Tooltip location, polished example spacing and documentation on PersonaPage.",
+      "type": "patch"
```
Comment (mtennoe) [inline common/changes/office-ui-fabric-react/person-edge-polish_2017-11-06-16-27.json]: Considering this is adding a new capability to the Persona component, I would consinder this a minor version, not a patch

File: packages/office-ui-fabric-react/src/components/Persona/Persona.scss [hunk 4]
```diff
@@ -219,6 +220,17 @@ $ms-Persona-presenceBorder: 2px;
   }
 }
 
+.root.coinSizeWithPresence {
+  .presence {
+    height: $ms-Persona-presenceSizeFlex;
```
Comment (mtennoe) [inline packages/office-ui-fabric-react/src/components/Persona/Persona.scss]: This gets overridden by the other sizes if they are set right?

File: packages/office-ui-fabric-react/src/components/Persona/Persona.scss (hunk 2 above)
Comment (lynamemi) [reply packages/office-ui-fabric-react/src/components/Persona/Persona.scss]: There wasn't a quick fix to get the icon to display at the correct scale for any size so the intention was to hide it for now (it was super tiny and in the wrong location). I talked to some folks in the office just now and it sounds like the custom coin size should be a large conversation anyway, especially if we need to go further than the quick fix route. I'll pull this out into a separate PR.

File: packages/office-ui-fabric-react/src/components/Persona/Persona.tsx (hunk 1 above)
Comment (lynamemi) [reply packages/office-ui-fabric-react/src/components/Persona/Persona.tsx]: In the ToolTip props documentation, it states that if directionalHintForRTL is not specified, a mirror of `directionalHint` will be used instead. I double checked the build, and it appears to be working as expected.
image

File: common/changes/office-ui-fabric-react/person-edge-polish_2017-11-06-16-27.json (hunk 3 above)
Comment (lynamemi) [reply common/changes/office-ui-fabric-react/person-edge-polish_2017-11-06-16-27.json]: With moving 'Flex sizing for presence with custom coinSize' to a separate PR, I think the rest of the changes would fall under the category of 'visual bug fixes', thus not requiring a minor version bump. What do you think?

File: packages/office-ui-fabric-react/src/components/Persona/Persona.tsx (hunk 1 above)
Comment (mtennoe) [reply packages/office-ui-fabric-react/src/components/Persona/Persona.tsx]: Cool, thanks!

**Pull request merged**